packages/manager/apps/public-cloud/src/components/beta-warning/index.js

```diff
@@ -0,0 +1,19 @@
+import angular from 'angular';
+import 'ovh-ui-angular';
+import ngOvhSidebarMenu from '@ovh-ux/ng-ovh-sidebar-menu';
+
+import component from './beta-warning.component';
+
+import './beta-warning.less';
+
+const moduleName = 'ovh-manager-betaWarning';
+
+angular
+  .module(moduleName, [
+    'oui',
+    ngOvhSidebarMenu,
+  ])
+  .component('betaWarning', component)
```

Comment:
As this component is "Public Cloud" oriented, what do you think about prefixing it?

For instance:

```suggestion
  .component('publicCloudBetaWarning', component)
```